subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/repositories/resolver/ExternalResourceResolver.java

```diff
@@ -325,7 +325,7 @@ private void publishPossiblyUnsupportedChecksum(ExternalResourceName destination
         try {
             publishChecksum(destination, content, algorithm, length);
         } catch (Exception ex) {
-            LOGGER.warn("Cannot upload checksum for " + content.getName() + ". Remote repository doesn't support " + algorithm + ". Error: " + ex.getMessage());
+            LOGGER.warn("Cannot upload checksum for " + content.getName() + ". Remote repository doesn't support " + algorithm + ". Error: " + ex.getMessage() + ". This will not fail this build");
```

Comment:
I think we could remove the error completely, since it's non-fatal:

```suggestion
            LOGGER.warn("Cannot upload checksum for " + content.getName() + " because the remote repository doesn't support " + algorithm + ". This will not fail the build.");
            LOGGER.info("Upload error", e);
```

WDYT?

Reply:
that makes sense to me 👍